ReleaseHistory.md

```diff
@@ -15,6 +15,25 @@
 * PRF: Add support for efficiently peeking into non-seekable streams for binary/text categorization.
 * NEW: Add a new `--timeout-in-seconds` parameter to `AnalyzeOptionsBase`, which will override the `TimeoutInMilliseconds` property in `AnalyzeContextBase`.
 * NEW: `--post-uri` will skip sending the SARIF log to the configured endpoint if the file contains no results or fatal execution errors.
+* NEW: Add the following rules:  
+  `ADO1011.ReferenceFinalSchema`,  
+  `ADO1013.ProvideRequiredSarifLogProperties`,  
+  `ADO1014.ProvideRequiredRunProperties`,  
+  `ADO1015.ProvideRequiredResultProperties`,  
+  `ADO1016.ProvideRequiredLocationProperties`,  
+  `ADO1017.ProvideRequiredPhysicalLocationProperties`,  
+  `ADO1018.ProvideRequiredToolProperties`,  
+  `ADO2008.ProvideRequiredSarifLogProperties`,  
```

Comment:
@EasyRhinoMSFT 
I found a problem here, we have 2 ProvideRequiredSarifLogProperties for Ado.

![image](https://github.com/microsoft/sarif-sdk/assets/81775155/38f0f885-c57c-4b33-9e2f-b30611a20716)
Do we have duplicated rule, or the file name is wrong.
The .cs file name must be consistent with the rule name.

Reply:
Looking...

Reply:
We can get rid of 2008, it's covered by 1013 (plus a bunch more)

Reply:
Got it, delete should be easy to do, I will delete it.

Reply:
deleted.
Added a todo